db/seeds/test_subscribers.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+exports.seed = function(knex, Promise) {
```

Comment:
Not sure if `Promise` is used or if we can remove it. It may be confusing naming given the native `Promise` object/class.

Reply:
Good catch. This was auto-generated by knex. Removing the `Promise` param.